### PR TITLE
feat: add gemini code sample and test for text-only input

### PR DESCRIPTION
**Pull request by irataxy**
## Description

Fixes b:336785030

- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java
Comment (Sita04) [inline vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java:54]: Would it okay to make this description a little more elaborate?
Unsure of what "analyzes" refers to here.

**Review by Sita04 — APPROVED** [review]
Thanks, LGTM.
Will wait for Eric's approval to merge.

File: vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java
Comment (irataxy) [reply vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java:20]: Yes, this is a citation for the movie review.

File: vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java
Comment (irataxy) [reply vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java]: Done

File: vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java
Comment (irataxy) [reply vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java]: Thanks! Should be fixed now.

**Review by gericdong — APPROVED** [review]

**Pull request merged**
